v2/emailpassword/common-customizations/get-user-info.mdx

```diff
@@ -123,15 +138,15 @@ user_info = ^{getUserByEmailPython}("public", "test@example.com")
 </TabItem>
 </BackendSDKTabs>
 
-
 <CustomAdmonition type="multi-tenancy">
 
-Notice that the first argument of the above function is `"public"`. This is the default tenantId which means that SuperTokens will return information about the user whose email is `"test@example.com"` in the `"public"` tenant. 
+Notice that the first argument of the above function is `"public"`. This is the default `tenantId` which means that SuperTokens will return information about the user whose email is `"test@example.com"` in the `"public"` tenant. 
 
-If you are using our multi tenancy feature, you can pass in a different tenantId to get information about a user in a different tenant.
+If you are using our multi-tenancy feature, you can pass in a different `tenantId` to get information about a user in a different tenant.
 
 </CustomAdmonition>
 
+<!-- END COPY SECTION -->
```

Comment:
there is a section below that says "Using the `getUserById` function". But in node js, the function is not called getUserById (whereas in golang and python, it is). So that needs fixing too. Please go through the whole page and double check all snippets

Reply:
I reworked that section to reference the specific functions for each language.